core/api/src/services/notifications/index.ts

```diff
@@ -194,47 +199,34 @@ export const NotificationsService = (): INotificationsService => {
     transaction,
   }: NotificatioSendTransactionArgs): Promise<true | NotificationsServiceError> => {
     try {
-      const settings = await getUserNotificationSettings(recipient.userId)
-      if (settings instanceof Error) {
-        return settings
-      }
-
-      const hasDeviceTokens = settings.pushDeviceTokens.length > 0
-      if (!hasDeviceTokens) return true
+      const request = new HandleNotificationEventRequest()
+      const event = new NotificationEvent()
+      const tx = new TransactionInfo()
 
+      tx.setUserId(recipient.userId)
       const type = getPushNotificationEventType(transaction)
       if (!type) return true
-
+      tx.setType(notificationTypeToProto(type))
+      const settlementAmount = new ProtoMoney()
+      settlementAmount.setMinorUnits(transaction.settlementAmount)
+      settlementAmount.setCurrencyCode(transaction.settlementCurrency)
+      tx.setSettlementAmount(settlementAmount)
       const displayAmountMajor = transaction.settlementDisplayAmount
       const displayCurrency = transaction.settlementDisplayPrice.displayCurrency
-      const displayAmountMinor = roundToBigInt(
-        majorToMinorUnit({ amount: Number(displayAmountMajor), displayCurrency }),
+      const displayAmountMinor = 
+        Math.round(majorToMinorUnit({ amount: Number(displayAmountMajor), displayCurrency }))
+      const displayAmount = new ProtoMoney()
+      displayAmount.setMinorUnits(displayAmountMinor)
+      displayAmount.setCurrencyCode(displayCurrency)
+      tx.setDisplayAmount(displayAmount)
+      event.setTransaction(tx)
+      request.setEvent(event)
+
+      const response = await notificationsGrpc.handleNotificationEvent(
```

Comment:
not a blocker but the method name looks more like the name for a callback handler.... maybe something like `sendEvent` is enough

Reply:
well... we've introduced this name already in many places (KYC, Circles). The thing is `send` is not accurate as the question wether or not to send happens in notifications...